### ISSUE TITLE
When submitting for IOS review this error is now appearing - PrivacyInfo.xcprivacy

### ISSUE THREAD

**Issue by samargy** [eopeter/flutter_mapbox_navigation#357]
TMS-91056: Invalid privacy manifest - The PrivacyInfo.xcprivacy file from the following path is invalid: “Frameworks/MapboxCommon.framework/PrivacyInfo.xcprivacy”. Keys and values in any privacy manifest must be in a valid format. For more details about privacy manifest files, visit: ....

MapboxCore and MapBoxNavigation both released fixes to this in their 2.18 versions

**Comment by sureshimp766** [comment]
yes we are getting same error this week and it is stopping us from uploading the app

**Comment by jiraya85** [comment]
Any update on this?
